src/setup/defaultNodes.js

```diff
@@ -42,6 +43,7 @@ const nodeTypes = [
   association,
   pool,
   poolLane,
+  boundaryMessageEvent,
```

Comment:
Remove this from here and place it in the `timerEventNodes` array.

Elements in `nodeTypes` are parsed based on their BPMN type, however, the BPMN type for all the different boundary events is `bpmn:BoundaryEvent`, so the parsing of boundary events was  configured to handle this. The name, `timerEventNodes`, isn't really accurate anymore, maybe you can re-name it to something like `timerAndBoundaryEventNodes.`